geth/testing/testing.go

```diff
@@ -88,15 +102,33 @@ func (s *BaseTestSuite) StartTestNode(networkID int, opts ...TestNodeOption) {
 	require.True(s.NodeManager.IsNodeRunning())
 }
 
-// TestNodeOption is a callback passed to StartTestNode which alters its config.
-type TestNodeOption func(config *params.NodeConfig)
+// ResetSyncFlag resets the nodeSyncCompleted flag.
+func (s *BaseTestSuite) ResetSyncFlag() {
+	s.nodeSyncCompleted = false
+}
```

Comment:
It's also used in a single test with a clearly-defined purpose and unlike to be used anywhere else. We seem to better not to have this function.

Reply:
Alright